tests/console/hostname.pm

```diff
@@ -22,6 +22,7 @@ sub run() {
 
     script_run "hostname";
     assert_screen("hostname-$hostname");
+    assert_script_run "systemctl -q is-active network.service && systemctl restart network.service";
```

Comment:
Do you require the test or does it harm? If it does harm without your modifications, I would prefer to just skip it in the HA tests instead of doing something rather unnecessary in all scenarios. 

I'm mixed if what you do will hide bugs in other scenarios or even trigger new problems or both.